guides/source/development_dependencies_install.md

```diff
@@ -46,8 +46,7 @@ $ cd rails
 
 The test suite must pass with any submitted code. No matter whether you are writing a new patch, or evaluating someone else's, you need to be able to run the tests.
 
-Install first SQLite3 and its development files for the `sqlite3` gem. Mac OS X
-users are done with:
+Install first SQLite3 and its development files for the `sqlite3` gem. MacOS users are done with:
```

Comment:
We should certainly not change the case here. It feels a bit weird to start a sentence with a lower case letter but you can rephrase it like:

> On macOS you are done with

Could you also please wrap this line at 80 chars ? :-)